webviz_subsurface/plugins/_tornado_plotter_fmu/views/plot_view/_plot_view.py

```diff
@@ -1,19 +1,211 @@
-from webviz_config.utils import StrEnum
-from webviz_config.webviz_plugin_subclasses import ViewABC
+import json
+import sys
+from dataclasses import dataclass
+from typing import List, Optional, Union
 
+import pandas as pd
+import webviz_core_components as wcc
+from dash import Input, Output, callback
+from dash.development.base_component import Component
+from dash.exceptions import PreventUpdate
+from webviz_config.utils import StrEnum, callback_typecheck
+from webviz_config.webviz_plugin_subclasses import SettingsGroupABC, ViewABC
+
+from webviz_subsurface._components.tornado._tornado_bar_chart import TornadoBarChart
+from webviz_subsurface._components.tornado._tornado_data import TornadoData
+
+from ...shared_settings import FilterOption, Scale
 from .view_elements import TornadoPlot
 
+if sys.version_info >= (3, 9):
+    from typing import Annotated
+else:
+    from typing_extensions import Annotated
+
+
+class PlotOption(StrEnum):
+    SHOW_REAL_POINTS = "show-real-points"
+    COLOR_BARS_BY_SENS = "color-bars-by-sens"
+
+
+class LabelOption(StrEnum):
+    SIMPLE = "simple"
+    DETAILED = "detailed"
+    HIDE = "hide"
+
+
+class PlotSettings(SettingsGroupABC):
+    class IDs(StrEnum):
+        PLOT_OPTIONS = "plot-options"
+        LABEL = "label"
+
+    def __init__(
+        self,
+    ) -> None:
+        super().__init__("Plot settings")
+        self.plot_options = [
+            {
+                "label": "Show realization points",
+                "value": PlotOption.SHOW_REAL_POINTS,
+            },
+            {
+                "label": "Color bars by sensitivity",
+                "value": PlotOption.COLOR_BARS_BY_SENS,
+            },
+        ]
+        self.label_options = [
+            {"label": "No label", "value": LabelOption.HIDE},
+            {
+                "label": "Simple label",
+                "value": LabelOption.SIMPLE,
+            },
+            {
+                "label": "Detailed label",
+                "value": LabelOption.DETAILED,
+            },
+        ]
```

Comment:
Consider dropping attributes and define the list of options directly in the `layout`-function below? If not make them private with prefix `_` - i.e. `self._plot_options` etc.